### ISSUE TITLE
Warning : DateTime::setTimestamp() expects parameter 1 to be integer, float given

### ISSUE THREAD

**Issue by novhex** [bitpay/php-bitpay-client#256]
Hello, i have downloaded the latest version of php-bitpay-client and i successfully configured the settings then when i tested it to create an invoice , i encountered this warning message

```
Warning: DateTime::setTimestamp() expects parameter 1 to be integer, float given in C:\xampp\htdocs\bitpaydemo\php-bitpay-client-master\src\Bitpay\Invoice.php on line 477

Warning: DateTime::setTimestamp() expects parameter 1 to be integer, float given in C:\xampp\htdocs\bitpaydemo\php-bitpay-client-master\src\Bitpay\Invoice.php on line 502

Warning: DateTime::setTimestamp() expects parameter 1 to be integer, float given in C:\xampp\htdocs\bitpaydemo\php-bitpay-client-master\src\Bitpay\Invoice.php on line 527 
```

but however the invoice id is generated . what could be the problem?

**Comment by pieterpoorthuis** [comment]
Can you show me the PHP code where you create the invoice?

**Comment by pieterpoorthuis** [comment]
Fixed in v2.2.16
